### PR TITLE
Feature: add location option to create menus in QGIS browser from project

### PR DESCRIPTION
**Pull request by jmkerloch**
Related to #107 #128 

First implementation of QGIS browser use to display project.

Missing:

- [ ] configuration of project location to add browser (waiting for #127)

For this a refactoring of layer load to use a list of layer configuration for Add all option instead of using actions available in menu. This couldn't be used in the QGIS browser.

### CODE REVIEW

File: menu_from_project/ui/browser.py
Comment (Guts) [inline menu_from_project/ui/browser.py:1]: I suggest to use a name less generic to prevent conflicts with common python or plugins modules.

**Comment by Guts** [comment]
Thanks for your work on this.

So the PR does not ship any UI widget to configure the menu location (menu, browser, new layer) right?

**Comment by jmkerloch** [comment]
> Thanks for your work on this.
> 
> So the PR does not ship any UI widget to configure the menu location (menu, browser, new layer) right?

For now there is no configuration for the browser. I'm waiting for the new project configuration UI.
Keeping PR as Draft until that.

File: menu_from_project/ui/browser.py
Comment (jmkerloch) [reply menu_from_project/ui/browser.py:1]: Updated to `menu_layer_data_item_provider`

**Comment by jmkerloch** [comment]
@Guts  I just added the option to add project in QGIS browser.

We also support merge of project, but  :warning:  currently we have no way of keeping the order of the layer so the layer will be mixed between project.

**Comment by Guts** [comment]
> We also support merge of project, but ⚠️ currently we have no way of keeping the order of the layer so the layer will be mixed between project.

Can you add a documentation section or page with the new option, including this limitation please?

**Review by Guts — APPROVED** [review]
LGTM :rocket: !

Thanks for taking in account all my remarks and concerns :pray:.

**Pull request merged**